test/system-test/test-config/core_pkcs11_config.h

```diff
@@ -51,7 +51,7 @@
 #endif
 
 #ifndef LIBRARY_LOG_LEVEL
-    #define LIBRARY_LOG_LEVEL    LOG_DEBUG
+    #define LIBRARY_LOG_LEVEL    LOG_NONE
```

Comment:
Is logging being turned off for tests?

Reply:
It is enabled in CI through compile parameter though: https://github.com/FreeRTOS/corePKCS11/blob/main/.github/workflows/ci.yml#L23